From caffdc85c15deae6f5fbe2c13a4bec581e5b9829 Mon Sep 17 00:00:00 2001
Message-ID: <caffdc85c15deae6f5fbe2c13a4bec581e5b9829.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 17 Jan 2024 22:16:11 +0000
Subject: [PATCH 36/38] gnu: go-github.com-smartystreets-goconvey: Move to (gnu
 packages golang-check).

* gnu/packages/check.scm (go-github.com-smartystreets-goconvey): Move
from here ...
* gnu/packages/golang-check.scm: ... to here.

Change-Id: I19d8db6e3b07e1d165396c0498e328b70b2cc760
---
 gnu/packages/check.scm        | 25 -------------------------
 gnu/packages/golang-check.scm | 25 +++++++++++++++++++++++++
 2 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/check.scm b/gnu/packages/check.scm
index 5e82a09126..ba99501972 100644
--- a/gnu/packages/check.scm
+++ b/gnu/packages/check.scm
@@ -898,31 +898,6 @@ (define-public python-gixy
 command.")
       (license license:mpl2.0))))
 
-(define-public go-github.com-smartystreets-goconvey
-  (package
-    (name "go-github.com-smartystreets-goconvey")
-    (version "1.6.3")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/smartystreets/goconvey")
-                    (commit version)))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1ph18rkl3ns3fgin5i4j54w5a69grrmf3apcsmnpdn1wlrbs3dxh"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/smartystreets/goconvey"))
-    (propagated-inputs
-     (list go-github.com-jtolds-gls go-github.com-smartystreets-assertions))
-    (synopsis "Go testing tool with both a web and terminal user interface")
-    (description
-     "GoConvey is a testing tool for Go. It integrates with go test, can show
-test coverage and has a web user interface that will refresh automatically.")
-    (home-page "https://github.com/smartystreets/goconvey")
-    (license license:expat)))
-
 (define-public googletest
   (package
     (name "googletest")
diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 0c37849806..c7b6e94791 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -412,6 +412,31 @@ (define-public go-github.com-smartystreets-assertions
 functions for writing tests in Go.")
     (license license:expat)))
 
+(define-public go-github.com-smartystreets-goconvey
+  (package
+    (name "go-github.com-smartystreets-goconvey")
+    (version "1.6.3")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/smartystreets/goconvey")
+             (commit version)))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1ph18rkl3ns3fgin5i4j54w5a69grrmf3apcsmnpdn1wlrbs3dxh"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/smartystreets/goconvey"))
+    (propagated-inputs
+     (list go-github.com-jtolds-gls go-github.com-smartystreets-assertions))
+    (home-page "https://github.com/smartystreets/goconvey")
+    (synopsis "Go testing tool with both a web and terminal user interface")
+    (description "GoConvey is a testing tool for Go. It integrates with go
+test, can show test coverage and has a web user interface that will refresh
+automatically.")
+    (license license:expat)))
+
 (define-public go-github.com-smartystreets-gunit
   (package
     (name "go-github.com-smartystreets-gunit")
-- 
2.41.0

